x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_definition.tsx

```diff
@@ -0,0 +1,263 @@
+                    </EuiButtonEmpty>
+                  ) : (
+                    <EuiText size="s">{getRuleConditionsWording()}</EuiText>
+                  )}
+                </EuiFlexGroup>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiFlexGroup>
+              <ItemTitleRuleSummary>
+                {i18n.translate('xpack.triggersActionsUI.ruleDetails.runsEvery', {
+                  defaultMessage: 'Runs every',
+                })}
+              </ItemTitleRuleSummary>
+
+              <ItemValueRuleSummary
+                data-test-subj="ruleSummaryRuleInterval"
+                itemValue={formatInterval(rule.schedule.interval)}
+              />
+            </EuiFlexGroup>
+
+            <EuiSpacer size="m" />
+
+            <EuiFlexGroup>
+              <ItemTitleRuleSummary>
+                {i18n.translate('xpack.triggersActionsUI.ruleDetails.notifyWhen', {
+                  defaultMessage: 'Notify',
+                })}
+              </ItemTitleRuleSummary>
+              <ItemValueRuleSummary itemValue={String(getNotifyText())} />
+            </EuiFlexGroup>
+
+            <EuiSpacer size="m" />
+            <EuiFlexGroup alignItems="baseline">
+              <ItemTitleRuleSummary>
+                {i18n.translate('xpack.triggersActionsUI.ruleDetails.actions', {
+                  defaultMessage: 'Actions',
+                })}
+              </ItemTitleRuleSummary>
+              <EuiFlexItem grow={3}>
+                <RuleActions ruleActions={rule.actions} actionTypeRegistry={actionTypeRegistry} />
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiPanel>
+      {editFlyoutVisible && (
+        <RuleEdit
+          onSave={() => {
+            setEditFlyoutVisible(false);
+            return onEditRule();
+          }}
+          initialRule={rule}
+          onClose={() => setEditFlyoutVisible(false)}
+          ruleTypeRegistry={ruleTypeRegistry}
+          actionTypeRegistry={actionTypeRegistry}
+        />
+      )}
+    </EuiFlexItem>
+  );
+};
+
+export interface ItemTitleRuleSummaryProps {
+  children: string;
+}
+export interface ItemValueRuleSummaryProps {
+  itemValue: string;
+  extraSpace?: boolean;
+}
+
+function ItemValueRuleSummary({
+  itemValue,
+  extraSpace = true,
+  ...otherProps
+}: ItemValueRuleSummaryProps) {
+  return (
+    <EuiFlexItem grow={extraSpace ? 3 : 1} {...otherProps}>
+      <EuiText size="s">{itemValue}</EuiText>
+    </EuiFlexItem>
+  );
+}
+
+function ItemTitleRuleSummary({ children }: ItemTitleRuleSummaryProps) {
+  return (
+    <EuiTitle size="xxs">
+      <EuiFlexItem style={{ whiteSpace: 'nowrap' }} grow={1}>
+        {children}
+      </EuiFlexItem>
+    </EuiTitle>
+  );
+}
+
+export type TimeUnitChar = 's' | 'm' | 'h' | 'd';
+
+const formatInterval = (ruleInterval: string) => {
```

Comment:
nitpick: I saw this came from a util file originally, I'm wondering if this will get reused later? If so maybe we can put it into a utils file again and write some unit tests.